package.json

```diff
@@ -32,6 +33,7 @@
         "react-dom": "17.0.2",
         "react-markdown": "^8.0.1",
         "react-syntax-highlighter": "^15.4.5",
+        "squeak-js": "^0.1.1",
```

Comment:
This won't actually work until https://github.com/PostHog/squeak-snippet/pull/3 is merged